src/map_name_popup.c

```diff
@@ -177,6 +188,120 @@ static const u8 sRegionMapSectionId_To_PopUpThemeIdMapping[] =
     [MAPSEC_TRAINER_HILL - KANTO_MAPSEC_COUNT] = MAPPOPUP_THEME_MARBLE
 };
 
+// gen 5 assets
```

Comment:
I'm not a 100% sure on this, but I think it would be preferable to surround these with #if referring to the configs to avoid including unused assets in the build

Reply:
I started with that, but I pivoted away from the "going nuts with preprocs" approach I started with on psf's recommendation. Preproc removing the BW assets would mean I have to preproc out where they're referenced later in the file, which would only be a few lines so I guess that wouldn't be too bad.

Or could do something like this maybe...but it is kinda cursed.

```c
#if MAP_POPUP_GENERATION == GEN_5
// Gen5 assets
static const u8 sMapPopUpTilesPrimary_BW[] = INCBIN_U8("graphics/map_popup/bw/bw_primary.4bpp");
static const u8 sMapPopUpTilesSecondary_BW[] = INCBIN_U8("graphics/map_popup/bw/bw_secondary.4bpp");
static const u16 sMapPopUpTilesPalette_BW_Black[16] = INCBIN_U16("graphics/map_popup/bw/black.gbapal");
static const u16 sMapPopUpTilesPalette_BW_White[16] = INCBIN_U16("graphics/map_popup/bw/white.gbapal");
#else
static const u8 sMapPopUpTilesPrimary_BW[] = {0};
static const u8 sMapPopUpTilesSecondary_BW[] = {0};
static const u16 sMapPopUpTilesPalette_BW_Black[16] = {0};
static const u16 sMapPopUpTilesPalette_BW_White[16] = {0};
#endif
```